train.py

```diff
@@ -120,6 +120,11 @@ def train_gpu(rank,world_size,opt,dataset):
                     if opt.display_id > 0:
                         accuracies=model.get_current_D_accuracies()
                         visualizer.plot_current_D_accuracies(epoch, float(epoch_iter) / dataset_size, accuracies)
+                if opt.adaptive_pseudo_augmentation and total_iters % opt.APA_every < batch_size:
```

Comment:
We may want this into `base_model.py` instead, no ?

Reply:
Maybe it would be more logiacal I don't know actually.

Reply:
But we currently don't have `total_iters` in `base_model.py`, it's why I chose to update this in `train.py
`